run.sh

```diff
@@ -80,6 +83,7 @@ case "$deployment_action" in
         deploy_airship
         ;;
     "setup_everything")
+	deploy_network
```

Comment:
We should also add this to "setup_hosts", right? And maybe we should be calling "setup_hosts" in "setup_everything" instead of repeating the same list of tasks?

Reply:
Agreed on the fact to add it into setup_hosts (in other words not expose it to end users). (See comment above).

For the setup_everything, I kinda like your idea. Want to propose a PR for that?

Reply:
@jesusaurus added it to `setup_hosts` step.
In terms of `setup_everything` - I think we should try to split between
a) the setup of the infrastructure (network, servers/volumes for caasp, ceph, installation of caasp, ceph) 
b) the actual installation on a given environment

A customer will not install this on openstack or libvirt but (I assume) on a already deployed caasp and ceph cluster. So the steps that do that deployment (on a given env) are imo more important (and customer facing) than the other stuff (which is nice for demos, testing, devs, CI, ...)

Reply:
@toabctl I agree that we should maintain a logical split between (a) infrastructure setup, and (b) the actual installation; but `setup_everything` already blurs that line. My understanding is that we have `setup_hosts` for (a) and then either `setup_airship` or `setup_openstack` for (b); and that `setup_everything` is equivalent to calling `setup_hosts` followed by `setup_openstack`.